### PR TITLE
Add support for verifying/testing solutions

### PR DESCRIPTION
**Pull request by darcyclarke**
Added some `npm scripts` to be able to pull down/test solutions. Had to rename the files from `index.js` to make my life less of a hell and also found out that `learnyounode` hasn't been updated in a year on **npm** which made me nuts because I couldn't figure out why this handy repo ([workshopper-adventure-test](https://github.com/workshopper/workshopper-adventure-test)) had a different version whenever I installed `learnyounode` then from when I cloned the repo (ie. I went with checking out the repo and then doing an `npm i` within that cloned repo vs. a straight dep, or else the tests wouldn't run... it was annoying)

Anyways...

You now have:
- `npm i` - installs the latest `learnyounode` repo locally, /w dev deps
- `npm run build` - runs ricky's nice readme generation along with moves **solution** files into the corresponding workshopper
- `npm run test` - runs the `workshopper-adventure-test` cli tool against the modified/updated files inside the workshopper
- `npm run clean` - to remove the `tmp` directory that we store the repo in 

Enjoy!

### CODE REVIEW

**Comment by mikemimik** [comment]
LGTM

**Review by JasonEtco — APPROVED** [review]
:shipit: 🚢 :octocat:

**Comment by darcyclarke** [comment]
@rickycodes Ya... looking at this without be tired... I can see a ton of improvements. If you want to switch to `npm i {repo}` that would be wise and then you can just get rid of the `clean` script and modify the other build/test as necessary. Should be pretty straightforward and I think, because that CLI tool uses mocha under the hood, then hooking this up to Travis should be a breeze.

**Review by mikemimik — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "learnyounode-golf",
+  "version": "1.0.0",
+  "description": "my attempt at solving [`learnyounode`](https://github.com/workshopper/learnyounode) with the least amount of code possible",
+  "scripts": {
+    "build": "node ./scripts/build.js",
+    "clean": "rm -r -f ./tmp/",
+    "install": "[ -e ./tmp/ ] || (git clone git@github.com:workshopper/learnyounode.git ./tmp && cd ./tmp && npm i)",
+    "test": "npm run install && npm run build && ./tmp/node_modules/.bin/workshopper-adventure-test ./tmp/"
```
Comment (rickycodes) [inline package.json:9]: 🤔 I don't think this actually tests the solutions themselves, this just runs a test on the workshopper?

**Comment by rickycodes** [comment]
closing in favor of https://github.com/rickycodes/learnyounode-golf/pull/11

**Pull request closed without merging**